app/src/main/java/org/mifos/selfserviceapp/ui/activities/HomeActivity.java

```diff
@@ -42,6 +49,12 @@
     @Inject
     PreferencesHelper preferencesHelper;
 
+    @Inject
+    UserDetailsPresenter detailsPresenter;
+
+    TextView username;
```

Comment:
I'd suggest suffixing class identifier in case of views, so it becomes more readable in later part of the code.

Something like ```tvUsername``` and ```ivUserProfilePicture``` etc.

And also the views can be made ```private``` unless you have a plan to access them from outside this class.

What do you think?